frontend/src/router.js

```diff
@@ -93,6 +93,20 @@ export default new Router({
       },
       beforeEnter: requireAuth
     },
+    {
+      path: '/camps/invitation/:inviteKey',
+      name: 'campInvitation',
+      components: {
+        navigation: NavigationAuth,
+        default: () => import(/* webpackChunkName: "login" */ './views/camp/Invitation')
+      },
+      props: {
+        default: route => ({
+          getCampCollaboration: campCollaborationFromInviteKey(route.params.inviteKey),
+          getCamp: campFromInviteKey(route.params.inviteKey)
```

Comment:
Interesting that you do this here. I'd only resolve dependencies via the router if it's absolutely necessary to make sure the data is plausible and can be displayed. In the case of the camp here, I'd just compute that from the campCollaboration directly in Invite.vue, since the URL doesn't contain the camp id.

Reply:
habe es so gelöst, dass die campCollaborations (Die Collection) jetzt als vuex-store proxy in die komponente hineinkommen,
und vo da die einzelne campCollaboration und das camp berechnet wird.
Dann brauch ich hier keinen expliziten State, sondern kann die Vue Magic mit dem Update der Komponente sobald sich eine property ändert verwenden.

Reply:
Now solved with RPC Controller, and we don't get a collection anymore